src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -74,6 +74,7 @@ public void ValidateConfigProperties()
         ValidateGlobalEndpointRouteConfig(runtimeConfig);
         ValidateAppInsightsTelemetryConnectionString(runtimeConfig);
         ValidateLoggerFilters(runtimeConfig);
+        ValidateAzureLogAnalyticsAuth(runtimeConfig);
```

Comment:
Why do we need to add an additional function ValidateAzureLogAnalyticsAuth if the json schema validation should be taking care of this validation for us already?